simple_graph.py

```diff
@@ -0,0 +1,60 @@
+class Graph(object):
+
+    def __init__(self):
+        self.graph = {}
+
+    def nodes(self):
+        l = []
+        for k, v in self.graph.iteritems():
+            l.append(k)
+        return l
+
+    def edges(self):
+        l = []
+        for k, v in self.graph.iteritems():
+            for edge in v:
+                l.append([k, edge])
```

Comment:
only complaint is about naming, is `edge` really an "edge" here?  It's just another node isn't it?